user-guide/Cloud_Platform/Catalog/About_the_Catalog_module.md

```diff
@@ -31,6 +31,14 @@ A Catalog item is a small package that contains one or more artifacts that can b
 
 Catalog items can have multiple versions. To make sure that the versioning of items is easy to understand for everyone, [Semver](https://semver.org/) versioning is enforced. Extra labels can be assigned to versions to indicate that a certain version is not an official release (e.g. 1.2.3-alpha).
 
+For Catalog items of type **Connector**, versions are grouped by **range**. A range is defined by the first three indicators of a version (e.g. **7.1.2**.33).
+
+The Catalog will recommend certain versions based on the following conditions:
+
+- The latest version of a range that is tagged as the "Main" version (visualized in green).
+- The latest version of a range that is tagged with a custom tag (visualized in gray).
+- If neither of the above apply, the latest version of the highest active range will be recommended.
+
```

Comment:
Could the additions I suggested in #3407 be included in this section instead? Or do you think they are OK in the location where I proposed to add them (though possibly with a link to this section)?

Reply:
Hmm, think we might want to move it here and then reference it.
Then we could have all the general information regarding the versioning on here.